content/css/jenkins.css

```diff
@@ -81,7 +81,7 @@ body h1.page-title .in-tag {
 }
 
 #doc-toolbar *,
-#ji-toolbar * {line-height:1.2rem !important; vertical-align:bottom !important;}
+#ji-toolbar * {line-height:1.5rem !important; vertical-align:bottom !important;}
```

Comment:
@recena I did it for a reason.The line height was different in the two popups(1-download button on banner and 2- download button in navbar ) that's why I changed the line height for consistency. If line height will be different then the height for a line for a paragraph will be different so adding a margin of 1.3125rem will not serve purpose at both popups.

I had added a comment that I was using a non-conventional way to align the buttons for quick fix that I will update later. Undoubtedly your solution is better.